### PR TITLE
SUSY 13TeV

### PR DESCRIPTION
**Pull request by alkaloge**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+<<<<<<< HEAD
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat]: Ah, there are git merge conflicts here somehow...

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+ 1d2  = etal    ! max rap for the charged leptons 
+ 0.6  = etaonium ! max rap for the quarkonium states
+   0  = etajmin ! min rap for the jets
+   0  = etabmin ! min rap for the b
+   0  = etaamin ! min rap for the photons
+   0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+<<<<<<< HEAD
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat]: and here...

Comment (alkaloge) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat]: I don’t know what is happening - This PR is from a fresh clean environment with a new brach… The previous PR was closed before I pushed the new one. What do you suggest ? In my area, I don’t see “conflicts"

Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat]: The conflict probably entered the file at an earlier stage, when performing some git commands in one of the previous areas.  (Ie these conflict resolution lines are present in the files which you've copied into the clean directory)

You should be able to simply fix them and add a commit.

Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat]: Ah, on the other hand, the problem could also be if you are reusing the same branch name as previously, you may be colliding with previous commits already in that branch when you do git push.

**Comment by alkaloge** [comment]
must be ok now...

**Pull request merged**